static/css/less/colors.less

```diff
@@ -74,6 +74,7 @@
 
 // greyscale
 @black: hsl(0, 0%, 0%);
+@lighter-black: hsl(0, 0%, 0%,.7);
```

Comment:
```suggestion
```
This new shade of grey is not needed.
According to the spec, the "Open" and "Closed" links should be green and red, respectively.  Whichever is currently selected should have a weight of 700, while the unselected link should have a weight of 400.  The hex values for the colors can be found in Figma, but you can use the following `hsl` values for each:
Green: `hsl(126, 100%, 30%)`
Red: `hsl(0, 100%, 50%)`

The same colors should be used for the dots found on the left-hand side of each table row.

Reply:
So i changed the links and dot accordingly and added these two colors.